post_processing/ci_plots.jl

```diff
@@ -475,6 +475,59 @@ function make_plots(
     )
 end
 
+function make_plots(
```

Comment:
Just a note that @Sbozzolo and I will fix the longrun output frequency soon, so this will not be needed. We can merge this for now.